packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2145,6 +2145,11 @@ class BuildOwner {
 
   int _debugStateLockLevel = 0;
   bool get _debugStateLocked => _debugStateLockLevel > 0;
+
+  //  Whether this element is currently in the process of getting built.
+  //
+  //  Only valid when asserts are enabled.
+  bool get debugBuilding => _debugBuilding;
```

Comment:
///

Reply:
This object is the BuildOwner, not an element. It should probably be "Whether this widget tree is in the build phase" or some such.